{
	"identifier": "CVE-2019-12761",
	"description": "A code injection issue was discovered in PyXDG before 0.26 via crafted Python code in a Category element of a Menu XML document in a .menu file. XDG_CONFIG_DIRS must be set up to trigger xdg.Menu.parse parsing within the directory containing this file. This is due to a lack of sanitization in xdg/Menu.py before an eval call.",
	"type": "network",
	"severity": "high",
	"state": "published",
	"datetime": "0000-00-00T00:00:00Z",
	"is_edited": false,
	"hardware": [],
	"software": [
		{
			"name": "pyxdg",
			"version": "\u003c 0:0.26-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bullseye"
		},
		{
			"name": "pyxdg",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-buster"
		},
		{
			"name": "pyxdg",
			"version": "\u003c 0.26-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bullseye"
		},
		{
			"name": "pyxdg",
			"version": "\u003c 0.26-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-sid"
		},
		{
			"name": "pyxdg",
			"version": "\u003c 0.26-1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "debian-bookworm"
		},
		{
			"name": "python3-xdg",
			"version": "\u003c 0:0.25-4ubuntu1.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-bionic"
		},
		{
			"name": "python-xdg",
			"version": "\u003c 0:0.25-4ubuntu1.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-bionic"
		},
		{
			"name": "pyxdg",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-cosmic"
		},
		{
			"name": "pyxdg",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-disco"
		},
		{
			"name": "pyxdg",
			"version": "any",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-eoan"
		},
		{
			"name": "pyxdg",
			"version": "\u003c 0.25-4ubuntu1.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-bionic"
		},
		{
			"name": "pyxdg",
			"version": "\u003c 0.25-4ubuntu0.14.04.1~esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "pyxdg",
			"version": "\u003c 0.25-4ubuntu0.16.04.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-xenial"
		},
		{
			"name": "python3-xdg",
			"version": "\u003c 0:0.25-4ubuntu0.14.04.1~esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "python-xdg",
			"version": "\u003c 0:0.25-4ubuntu0.14.04.1~esm1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-trusty"
		},
		{
			"name": "python3-xdg",
			"version": "\u003c 0:0.25-4ubuntu0.16.04.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-xenial"
		},
		{
			"name": "python-xdg",
			"version": "\u003c 0:0.25-4ubuntu0.16.04.1",
			"architecture": "any",
			"manager": "apt",
			"vendor": "ubuntu-xenial"
		}
	],
	"products": [
		{
			"name": "pyxdg",
			"version": "\u003c 0.26",
			"architecture": "any",
			"vendor": "python",
			"type": "software",
			"is_edited": false,
			"aliases": []
		}
	],
	"references": [
		"https://gist.github.com/dhondta/b45cd41f4186110a354dc7272916feba",
		"https://lists.debian.org/debian-lts-announce/2019/06/msg00006.html",
		"https://lists.debian.org/debian-lts-announce/2021/08/msg00003.html",
		"https://snyk.io/vuln/SNYK-PYTHON-PYXDG-174562",
		"https://ubuntu.com/security/notices/USN-4700-1",
		"https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-12761"
	],
	"weaknesses": [
		"CWE-94"
	]
}